src/WebServer.ino

```diff
@@ -2405,8 +2405,13 @@ void handle_devices() {
         if (Device[DeviceIndex].PullUpOption)
         {
           addFormCheckBox(F("Internal PullUp"), F("TDPPU"), Settings.TaskDevicePin1PullUp[taskIndex]);   //="taskdevicepin1pullup"
+          #if defined(ESP8266)
+          addFormNote(F("PullDown for GPIO-16 (D0)"));
```

Comment:
It looks like this does show this remark on all pin configurations, while it was previously only showing a remark when GPIO 16 was selected.
IMHO you only should show information that does matter to the active situation.

Reply:
Remark is always show to let you know that for GPIO-16 is PullDown instead of PullUp.

Reply:
Only show it when it is applicable.
If it currently isn't marked with a "attention" symbol, we can add that, but I would like to have the amount of information clutter limited to a minimum.
Doesn't mean it should not be mentioned, but just mention it when it is something for the user to take into account.
Or else we can also write which pins should be high on boot, which pins cannot be used for interrupts (pin 16) etc.

Reply:
OK, no problem